ReleaseProcessAutomation.Tests/Git/CommandLineGitClientTests.cs

```diff
@@ -508,13 +508,24 @@ public void CheckoutDiscard_CommitOneFileAndEdit_DoesNotContainEdit ()
   }
 
   [Test]
-  public void Tag_InitCommit_FindsTagAfterwards ()
+  public void Tag_CreatesTag_FindsTagWithProperMessageAfterwards ()
```

Comment:
Naming:
`Tag_WithTagNameAndMessage_CreatesTagWithProperMessage`

Operation - Parameters/Context (as needed) - Result/Expected operation

=> FindsTagAfterwards could imply that the Tag-operation does the finding. With the new name, it says that a tag should be created.